lib/sequel/adapters/shared/mssql.rb

```diff
@@ -476,6 +476,21 @@ def set_mssql_unicode_strings
         @mssql_unicode_strings = typecast_value_boolean(@opts.fetch(:mssql_unicode_strings, true))
       end
       
+
+      # Specify unicode strings by default for Strings.
```

Comment:
This section was generally copied from the generic version (`lib/sequel/database/schema_methods.rb`). I noticed that the pg adapter had a stanza that remapped the types similar to this.